Mathlib/Data/Set/Intervals/Image.lean

```diff
@@ -64,6 +64,7 @@ variable [Preorder α] [Preorder β]
 theorem Monotone.image_Icc_subset (h : Monotone f) :
     f '' Icc a b ⊆ Icc (f a) (f b) :=
   h.mapsTo_Icc.image_subset
+#align monotone.image_Icc_subset Monotone.image_Icc_subset
```

Comment:
Where are the other three lemmas, `MonotoneOn.image_Icc_subset`, `AntitoneOn.image_Icc_subset`, and `Antitone.image_Icc_subset`? I'm a bit confused, because your previous version had maybe 20 new lemmas, but this one is now missing 3 lemmas.

Reply:
Two approaches come to mind:
* Add `#noalign`s for the other three lemmas with a TODO comment
* Make a separate PR that adds this whole family of lemmas (first), then update this one to just add the aligns.

Reply:
Went for the second because it's less work for me: #7146.